cfme/tests/infrastructure/test_vmware_provider.py

```diff
@@ -514,3 +513,33 @@ def test_rebuilt_vcenter_duplicate_hosts(appliance, provider):
     # Using appliance.rest_api as hosts.all() do not return archived hosts, I need those too
     hosts_after = len(appliance.rest_api.collections.hosts.all)
     assert hosts_before == hosts_after
+
+
+@pytest.mark.tier(1)
+@pytest.mark.provider([VMwareProvider], selector=ONE_PER_TYPE, override=True)
+@pytest.mark.meta(automates=[1755070])
+def test_vm_notes_ui(appliance, provider, soft_assert):
+    """
+    Check if the VM Notes are shown in the CFME UI for VMs on VMware.
+
+    Bugzilla:
+        1755070
+
+    Polarion:
+        assignee: kkulkarn
+        casecomponent: Infra
+        caseimportance: medium
+        initialEstimate: 1/2h
+        testtype: functional
+        testSteps:
+            1.Add VMware provider to CFME
+            2.Navigate to Compute->Infrastructure->Virtual Machines
+            3.Select a VM to view details and click on "Container" in the "Basic Information" table
+            4.Check if Notes field exists and Notes are shown
+    """
+    vms_collections = appliance.collections.infra_vms
+    vm = vms_collections.instantiate(name='cu-24x7', provider=provider)
```

Comment:
Rather than hard-coding this vm name, I recommend doing 
```
    collection = provider.appliance.provider_based_collection(provider)
    vm = collection.all()[0]
```

Or since you're wanting to read the notes for this specific VM, you can use 
```    
pytest.mark.provider([VMwareProvider, RHEVMProvider], selector=ONE_PER_TYPE,
                         required_fields=[(['cap_and_util', 'capandu_vm'], 'cu-24x7')])
```

To ensure that the vm is on the provider from the yamls.

Reply:
https://github.com/ManageIQ/integration_tests/pull/9463/files/d64dc8f4100b0113152fe1c9c0927f6c16708958#diff-1dbb1b7d2fce29af647e11fa47b29ac6L29

Reply:
That link isn't working anymore because of commit re-writing, but I'm guessing you were linking to the module marker?

Override markers don't _add_ to the provider filter, they _define_ the provider filter for the local scope.

This needs to be fetched from yamls, and include the same `required_fields` filter applied at the module level.

Reply:
https://github.com/ManageIQ/integration_tests/blob/master/cfme/tests/infrastructure/test_vmware_provider.py#L29

Reply:
@mshriver updated the PR please check.